karma.conf.js

```diff
@@ -0,0 +1,65 @@
+// Karma configuration
+
+module.exports = function (config) {
+    config.set({
+
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: '',
```

Comment:
Didn't use a base path since if 'tests/javascript' is used it would be problematic when trying to access ones in 'media/system'.